src/file.cpp

```diff
@@ -68,7 +68,7 @@ namespace Sass {
         wchar_t wd[wd_len];
         wchar_t* pwd = _wgetcwd(wd, wd_len);
         if (pwd == NULL) throw Exception::OperationError("cwd gone missing");
-        sass::string cwd = wstring_to_string(pwd);
+        sass::string cwd = wstring_to_string(pwd).c_str();
```

Comment:
Why call `c_str()` here? Seems entirely redundant

Reply:
Also why do we have a `sass::string`?

Reply:
Because it's a specialization of std::string with the custom allocator.
Those can't be directly converted to std::string AFAICT.

Reply:
Strange, `basic_string` does define this https://en.cppreference.com/w/cpp/string/basic_string/operator%3D

Reply:
What error are you getting? This should just work.

Reply:
My guess is it's trying to call `Allocator(std::allocator const&)`, which we don't define. Defining this copy-constructor could then solve the incompatibility.

Reply:
Feel free to poke around once I get this to pass CI.